src/Conf.php

```diff
@@ -29,9 +29,9 @@ class Conf
     /**
      * Create new instance of Conf class.
      */
-    public function __construct()
+    public function __construct($filepath)
     {
-        $this->file = storage_path('app/conf.json');
+        $this->file = ($filepath) ? $filepath : storage_path('app/conf.json');
```

Comment:
@axelex I'm going to release tomorrow a new release with configurable file.
Just wait till tomorrow please.